### PR TITLE
Leaderboard: hit the API

### PR DESCRIPTION
**Pull request by WilhelmK109**
### In this milestone, I did the following:
- Created a new game with the name of my choice by using the API.
- Implemented the "Refresh" button that will get all scores for a game.
- Implemented the form "Submit" button that will save a score for a game.
- Used async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,55 +1,55 @@
...
+    const newName = inputName.value;
+    const newScore = Number(inputScore.value);
+    try {
+      await axios.post(`https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameId}/scores/`, {
+        user: newName,
+        score: newScore,
+      });
+      inputName.value = '';
+      inputScore.value = '';
+      refreshBtn.click();
```
Comment (victorokoroji) [inline src/index.js]: - [ ] KIndly remove the` refreshBtn` function you added after making the post request. Please note that when the post request is made, it shouldn't also fetch the data and display it on the UI. The data should only be displayed when you click on the refresh button 👍🏽

File: src/index.js [hunk 2]
```diff
@@ -1,55 +1,55 @@
...
+    } catch (error) {
+    // console.error(error);
```
Comment (victorokoroji) [inline src/index.js]: - [ ] Kindly remove all commented code from your codebase to make our code clean and readable 👍🏽

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @WilhelmK109, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the Git Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by franclobo — APPROVED** [review]
Hi @WilhelmK109, 

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️ 
- App works as expected 💯 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**